Documentation/operations/upgrade.rst

```diff
@@ -338,6 +338,8 @@ Helm Options
   ``encryption.secretName`` and ``encryption.interface`` are deprecated in
   favor of their ``encryption.ipsec.*`` counterparts and will be removed in
   Cilium 1.15.
+* Value ``hubble.peerService.enabled`` is deprecated and will be removed in
+  Cilium 1.15. The peer service is no longer optional.
```

Comment:
We already mention in [the v1.13 upgrade notes](https://docs.cilium.io/en/v1.13/operations/upgrade/#helm-options) that we intend to remove `hubble.peerService.enabled` in v1.14, so we could simply remove it in this PR.

Reply:
Right, the upgrade notes were in a backport commit and I missed it. Thanks! I updated the PR to remove the value directly.